scripts/Check-OfflineFilesList.ps1

```diff
@@ -0,0 +1,49 @@
+<# **
+  This script is part of Kiwix. It checks that the list of offline files precached by the Service Worker is complete
+  It is intended to be used primarily in CI testing, but can also be run from the commandline in open-source PS Core
+  https://github.com/PowerShell/PowerShell/releases/
+** #>
+
+# Provide the path and name of the Service Worker file relative to the Repository root
+$SWFile = 'service-worker.js'
+
+# List below any files, relative to Repository root, that are not required by the offline precache
+$ListOfExemptions = (
```

Comment:
Could you remind me why these files are exempted?

Reply:
Because these are files that are never used by our code. Maybe they are used as part of ancillary packaging for apps, for example, but will never actually be needed to run the app, so it would be a waste of valuable cache space (and download time) to load them, especially over slow bandwidth. If we don't exempt them, then the script will thrown an error, because these files are not loaded by the Service Worker.